### ISSUE TITLE
Optimization error: CalcCostFnInMatlab.mcr wrong CostFnPath

### ISSUE THREAD

**Issue by DamyHao** [korvin011/CSTMWS-Matlab-Interface#5]
Hi,
I have seen that the variable CostFnPath of the file CalcCostFnInMatlab.mcr points automatically to the Demo path and not to the default  MATLAB path. Could you write that on the docs? I spent hours trying to figure out why the optimization steps were not working.

Thanks

**Comment by korvin011** [comment]
Hi,
The `CostFnPath `written in the macro CalcCostFnInMatlab.mcr is determined in line [3732](https://github.com/korvin011/CSTMWS-Matlab-Interface/blob/8e3096b8d2c451a0797c6ca63269dd9894b579f5/Functions/TCSTInterface.m#L3732):
```
function SetMatlabCostFunction(this, CostFunction)
            ...
            [CostFnPath,CostFnName] = fileparts( TCSTInterface.GetFullPath(CostFunction) );
```
Perhaps, you had several functions named "CalcCostFn.m" in the MATLAB path, and perhaps the one from the demo appeared first. Indeed, such situations sometimes difficult to debug. To avoid such confusions, you can either give a unique name to the cost-function function, either specify a full path together with the function name in the CostFunction input variable.

**Comment by DamyHao** [comment]
Yes, my function has a unique name. The problem is that the code misses the last "\" and, in my case, I had to modify the CalcCostFnInMatlab.mcr manually to add the last "\" in the variable  	CostFnPath = "C:\Users\yourPath\MATLAB\\"
